enhancements/samples/removed-bootstrapping.md

```diff
@@ -0,0 +1,188 @@
+to prevent with this enhancement)
+2) Bootstrapping as `Removed` for a cluster where imagestreams are importable (i.e. breaking something that works today)
+
+Also note, a fair amount of tests in `e2e-*-builds`, `e2e-aws-image-ecosystem`, `e2e-aws-jenkins`, and the subset of 
+conformance tests in e2e-aws that leverage OpenShift Builds depend on the samples as a starting point.
+
+For e2e environments for IPv6 and disconnected / restricted network, those tests will either have to be filtered 
+out of the ginkgo query or reworked to either create their own imagestreams / templates or only leverage local
+resources that do not require network access (for example, binary builds).
+
+## Design Details
+
+### Test Plan
+
+Consider the following in developing a test plan for this enhancement:
+- QE already had some disconnected CI and we've been working with them via bugzillas to get it set up.  This change
+could remove the need for additional administration on their part to keep the cluster from having `Degraded` operators
+- For IPv6, the samples operator e2e will be updated to inspect the test pod for IPv6 and adjust accordingly
+
+### Graduation Criteria
+
+No tech preview, beta, multiple maturity levels needed.
+
+1) Samples install as they do traditionally in a connected IPv4 environment.
+2) Samples start as `Removed` in the special case scenarios noted above.
+
+Graduation will be achieved when this support GA's minimally in 4.4 and 4.3.z as part of IPv6 initiative. 
+
+### Upgrade / Downgrade Strategy
+
+Samples operator already identifies migration today, along with existing vs.
+new installations.
+
+The current plan is to only target new installations with these changes.
+
+However, an implementation where for an existing install we 
+
+- use our existing identifcation of whether imagestreap imports are failing
+- and the techniques noted above to discern whether disconnected/restricted network install or IPv6 single stack ar
+in play
+
+could allow us to take an existing installation with a samples operator either in `Degraded` state or destined to 
+be in `Degraded` state and adjust from `Managed` to `Removed` automatically.
+
+This choice is noted here for now to facilitate discussion during the enhancement review process.
+
+### Version Skew Strategy
+
+N/A
+
+## Implementation History
+
+## Drawbacks
+
+Working from `Removed` to a working `Managed` seems the easier / better path based on initial customer engagements with
+disconnected / restricted network install.  But Murphy's Law would probably say somebody has gotten used to the 
+existing out of the box behavior and would rather go from a broken `Managed` to functional `Managed` operator in
+one fell swoop.
+
+For example, as part of the multi-arch work, early adopters have requested that since samples exist out of the box for x86
+they should as well for s390x or ppc64le.
```

Comment:
to be fair, those adopters didn't ask for us to install content that would fail to import or otherwise fail to work, so i'm not sure how analogous the scenario is.  but yes it is indicative that there is an appetite for this content, so bootstrapping to removed may be viewed negatively by impacted users who, as you note above, may prefer to have "Broken managed content" than "no content", as their starting point.

Reply:
The `For example...` was added upon request from @adambkaplan in one of the earlier reviews.

Given that
- Perhaps I'm not capturing what he really wanted 
- Obviously I was originally fine with not mentioning this at all, since lack of it lead to him asking for it, and so I would not feel strongly about removing it if it is more trouble than it is worth.
- Or I could make it generic and say something along the lines of "samples are popular and not having them available and working out of the box will be viewed negatively"
- Or we can stand pat

WDYT @bparees ?

Reply:
> Or I could make it generic and say something along the lines of "samples are popular and not having them available and working out of the box will be viewed negatively"

personally i like this one, but we can see what @adambkaplan says.

it's not going to be a sticking point for me either way.